src/main/java/monitoringsystemturbo/presenter/MainPresenter.java

```diff
@@ -182,19 +182,36 @@ public void onAddApplication() {
     public void onAddActivity() {
         try {
             Application application = applicationListController.showActivityView();
+
             if (application != null) {
+
+                for (Application applicationFromList : loadedApplications) {
+                    if (applicationFromList.getName().equals(application.getName())) {
+                        List<Timeline> timelines = StatisticsManager.load(application.getName());
+                        TimelineElement timelineElement = new TimelineElement(application.getName(), timelines);
+                        timelineElement.setTimelineViewWidthByRegion(appTimelineContainer);
+
+                        TimelineElement timelineElement1 = timelineElements.get(application.getName());
```

Comment:
I dont like this name: timeLineElement1. It should indicate what it is for, now it isnt